x-pack/plugin/security/qa/security-basic/build.gradle

```diff
@@ -4,9 +4,8 @@ apply plugin: 'elasticsearch.java-rest-test'
 import org.elasticsearch.gradle.info.BuildParams
 
 dependencies {
-  javaRestTestImplementation project(path: xpackModule('core'))
```

Comment:
Other modules have kept this dependency - was it not required?

Reply:
There was an inconsistency in those dependencies by testArtifacts have provided that dependency before. We now ship this dependency on the main artifact as part of the testArtifacts (analog to what the gradle build in testFixture does). 

So when having a dependency on the `testArtifacts(project(path: xpackModule('core')))` we can consistently remove the explicit dependency on the main artifact. I'll go ahead and clean that up